include/rcutils/concat.h

```diff
@@ -22,6 +22,15 @@ extern "C"
 
 #include "rcutils/visibility_control.h"
 
+/// Return a newly allocated string that contains lhs, followed by delimiter, followed by rhs.
+/**
+ * This function allocates memory and returns it to the caller.  It is up to the
```

Comment:
each sentence should start on its own line to reduce coupling

Reply:
Thanks, I've fixed this here (and in `filesystem.h`).